project/phpunit.xml.dist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think you need to name it .twig because we are using twig here

Reply:
good catch

Reply:
wait, but README.md isn't named this way

Reply:
yes, some of them are using twig and not named .twig

Reply:
We could keep it this way and create a separate PR afterwards and rename all to `.twig`, but imo there are no advantages.

If we rename them, we don't see the different types in the idea like now:
![screenshot 2017-12-08 10 06 42](https://user-images.githubusercontent.com/995707/33758696-974c1358-dbff-11e7-9bae-567d2361cb9a.png)

Reply:
> yes, some of them are using twig and not named .twig

Not true. Both are different.

See: https://github.com/sonata-project/dev-kit/blob/13476204fd8eb4e03942084572dd1852eb7c8ae3/src/Console/Command/DispatchCommand.php#L497-L532

Reply:
ok, so not add `.twig` is fine in this case 👍